packages/docs/content/components/logos/healthcare-logo.mdx

```diff
@@ -30,6 +47,29 @@ import { Alert } from '@cmsgov/design-system';
     storyId="healthcare-logo--spanish-logo"
 />
 
+
+### Variations
+
+There are two variations of the HealthCare.gov identity marker that should be used sparingly. The first is a white logo against black. The second is a gray version. Both are intended to be used as "tags" in areas such as a footer where subtle use of the mark is appropriate.
+
+#### White logo on black
+<EmbeddedExample>
+<img src={blackLogoTag} />
+</EmbeddedExample>
+
+#### Grey logo
+<EmbeddedExample>
+<img src={greyLogoTag} />
+</EmbeddedExample>
```

Comment:
Since these are in example components with the option to view html, I was trying to figure out how to target the three classes inside to change the colors live like we do in the footer, but I haven't figured out a simple way of doing that besides creating new storybook stories. I feel like if we're going to make this an example, we should do the thing for real rather than show an image.